src/Sulu/Component/Webspace/Loader/XmlFileLoader11.php

```diff
@@ -54,10 +54,14 @@ protected function generateDefaultTemplates(Webspace $webspace)
             /* @var \DOMNode $node */
             $template = $node->nodeValue;
             $type = $node->attributes->getNamedItem('type')->nodeValue;
+            $parentTemplateNode = $node->attributes->getNamedItem('parent-template');
```

Comment:
Why did you handle that here in a different and more complicated way as opposed to `XmlFileLoader10`?

Reply:
XmlFileLoader was the one I was testing with, they are now equal. (11 had more checks for failure).